### ISSUE TITLE
Make site accessible

### ISSUE THREAD

**Issue by danielemery** [danielemery/leben-in-deutschland-test-practice#5]
An accessibility pass should be made over the UI. Particularly the display of if an answer was selected correctly or not. And aria tags in all the applicable places.
